build_library/grub_install.sh

```diff
@@ -71,6 +73,9 @@ cleanup() {
     if [[ -b "${LOOP_DEV}" ]]; then
         sudo losetup --detach "${LOOP_DEV}"
     fi
+    if [[ -e "${BUILD_LIBRARY_DIR}/grub.cfg" ]]; then
```

Comment:
This commit renames the default grub.cfg to grub-dev.cfg, and copies the correct variant for the image type to grub.cfg so that `tar` still works for memdisk creation. I strongly prefer your suggestion above.